### DISCUSSION TITLE
6.6.8

### DISCUSSION THREAD

**Discussion by kayhantolga** [betalgo/openai#111] [Announcements]
Breaking Changes

Renamed Engine keyword to Model in accordance with OpenAI's new naming convention.
Deprecated DefaultEngineId in favor of DefaultModelId.
DefaultEngineId and DefaultModelId is not static anymore.
Added support for Azure OpenAI, a big thanks to @copypastedeveloper!

Added support for Tokenizer, inspired by @dluc's https://github.com/dluc/openai-tools repository. Please consider giving the repo a star.

These two changes are recent additions, so please let me know if you encounter any issues.

Updated documentation links from beta.openai.com to platform.openai.com.

## What's Changed
* Add code of conduct 1 by @kayhantolga in https://github.com/betalgo/openai/pull/103
* created Contributing file by @kayhantolga in https://github.com/betalgo/openai/pull/104
* Dev by @kayhantolga in https://github.com/betalgo/openai/pull/105
* adding azure openai compatibility. by @copypastedeveloper in https://github.com/betalgo/openai/pull/107
* Tokinezer support by @kayhantolga in https://github.com/betalgo/openai/pull/108
* Feature/code clean up by @kayhantolga in https://github.com/betalgo/openai/pull/109
* Version 6.6.8 by @kayhantolga in https://github.com/betalgo/openai/pull/110

## New Contributors
* @copypastedeveloper made their first contribution in https://github.com/betalgo/openai/pull/107

**Full Changelog**: https://github.com/betalgo/openai/compare/v6.6.7...v6.6.8

<hr /><em>This discussion was created from the release <a href='https://github.com/betalgo/openai/releases/tag/v6.6.8'>6.6.8</a>.</em>

**Comment by copypastedeveloper** [comment]
I've started a conversation with folks at microsoft about getting you access so that you can continue to support their version of openai.  I'll let you know as I hear back.
